src/main/java/io/zksync/ethereum/EthereumProvider.java

```diff
@@ -60,6 +62,10 @@
         return contract.fullExit(BigInteger.valueOf(accountId), token.getAddress()).sendAsync();
     }
 
+    public CompletableFuture<TransactionReceipt> setAuthPubkeyHash(String pubKeyhash, AccountState state) {
```

Comment:
There should also be a function to check if the `authPubKeyHash` is set. Like this one:
https://github.com/matter-labs/zksync/blob/5b936b1855a08033cca7f75d6f87fde106c6e8fd/sdk/zksync.js/src/wallet.ts#L433